css/media.css

```diff
@@ -0,0 +1,388 @@
+@media (max-width: 1220px) {
+   .welcome-page {
+     width: 100%;
+     margin: 0 auto;
+   }
+ }
+
+ @media (max-width: 1036px) {
+   .header__menu_link {
+     font-size: 12px;
+     line-height: 15px;
+   }
+
+   .link__item {
+     font-size: 12px;
+     line-height: 15px;
+   }
+ }
+
+ @media (max-width: 856px) {
+   .header__menu_link {
+     display: none;
+   }
+
+   .welcome-page__speed-car {
+     width: 80%;
+   }
+
+   .welcome-page__speed-sec {
+     font-size: 28px;
+     line-height: 33px;
+   }
+
+   .link-none {
+     display: none;
+   }
+
+   .welcome-page__block {
+     width: 50%;
+     margin: 0;
+   }
+ }
+
+ @media (max-width: 768px) {
+   .header {
+     box-sizing: border-box;
+     padding: 24px 33px 0 15px;
+   }
+
+   .header__menu {
+     display: none;
+   }
+
+   .link-none {
+     display: none;
+   }
+
+   .link__img {
+     width: 35.23px;
+     height: 22.02px;
+     margin-left: auto;
+     margin-right: 0;
+   }
+
+   .car__title {
+     margin-top: 80px;
+   }
+
+   .car__subtitle {
+     position: relative;
+     bottom: 10px;
+     padding-left: 16px;
+   }
+
+   .car {
+     background-size: 109%;
+     background-position: 15% 50%;
+   }
+
+   .car__tsl {
+     width: 91%;
+     position: relative;
+     left: 10px;
+     top: 65px;
+   }
+
+   .welcome-page {
+     margin-top: 104px;
+     display: block;
+   }
+
+   .welcome-page__block{
+     width: auto;
+   }
+
+   .welcome-page__1 {
+     width: 100%;
+     display: flex;
+     justify-content: center;
+   }
+
+   .welcome-page__button-image {
+     width: 30px;
+     height: 30px;
+   }
+
+   .welcome-page__section {
+      margin-left: 0;
+      margin-right: 68px;
+   }
+
+   .welcome-page__speed-sec {
+     font-size: 50px;
+     line-height: 61px;
+   }
+
+   .welcome-page__driver-max {
+     font-size: 50px;
+     line-height: 61px;
+   }
+
+   .welcome-page__speed-car {
+     width: 120%;
+     padding: 18px 0 0 10px;
+   }
+
+   .discrip-mob {
+     margin-top: 2px;
+     margin-left: 38px;
+   }
+
+   .discrip-sek {
+     padding-left: 5px;
+     margin-bottom: 10px;
+     position: relative;
+     bottom: 8px;
+   }
+
+   .welcome-page__drive_img {
+     margin-right: 14px;
+     margin-top: 10px;
+   }
+
+   .km {
+     position: relative;
+     bottom: 7px;
+     left: 6px;
+   }
+
+   .welcome-page__driver-go {
+     margin-top: -2px;
+   }
+
+   .welcome-page__2 {
+     margin-top: 39px;
+     width: auto;
+     justify-content: center;
+     margin-left: 24px;
+   }
+
+   .welcome-page__test-drive {
+     font-size: 16px;
+     line-height: 20px;
+     color: #e51937;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)